src/types.ts

```diff
@@ -256,17 +280,17 @@ export interface Instance {
     /**
      * Returns the amount that zAuction has been approved to spend on behalf
      * of the given account.
-     * 
-     * Note: If all properties are null an attempt to get the legacy 
+     *
+     * Note: If all properties are null an attempt to get the legacy
      * zAuction balance is made instead. If that still returns nothing
-     * the user must call to approve 
-     * @param params 
-     * @param account The account
+     * the user must call to approve.
+     * @param account The account to check allowance for
+     * @param params A Bid object, tokenId, or the specific payment token
```

Comment:
It's technically not that, it's a `TokenAllowanceParams` object. This needs to have clear documentation around it's usage.